### PR TITLE
[FHB-117] Fix error page title not updating for postcode seach and letters pages

### PR DESCRIPTION
**Pull request by Top-Cat**

### CODE REVIEW

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/Shared/ProfessionalReferralCacheModel.cs [hunk 1]
```diff
@@ -22,9 +22,7 @@ protected ProfessionalReferralCacheModel(
         Errors = ErrorState.Empty;
     }
 
-    //todo: change to private set
-    //todo: remove this and reference Errors directly
-    public bool HasErrors { get; set; }
+    public bool HasErrors => Errors.HasErrors;
```
Comment (StuwiiDev) [inline src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/Shared/ProfessionalReferralCacheModel.cs:25]: The todo about referencing directly might be valid here, save having an extra property that just references the Errors.HasErrors

**Review by Zac-Digital — DISMISSED** [review]

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/Shared/ProfessionalReferralCacheModel.cs (hunk 1 above)
Comment (Top-Cat) [reply src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/Shared/ProfessionalReferralCacheModel.cs:25]: I read that todo as doing what I've done but happy to remove this variable instead.

**Review by Zac-Digital — APPROVED** [review]

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/ContactByPhone.cshtml.cs [hunk 2]
```diff
@@ -22,7 +22,7 @@ public ContactByPhoneModel(IConnectionRequestDistributedCache connectionRequestD
 
     protected override void OnGetWithModel(ConnectionRequestModel model)
     {
-        if (!HasErrors && model.ReferrerContact != null)
+        if (!Errors.HasErrors && model.ReferrerContact != null)
```
Comment (stevesatdfe) [inline src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/ContactByPhone.cshtml.cs:25]: Use model.ReferrerContact is not null perhaps?

**Review by stevesatdfe — APPROVED** [review]

File: src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/ContactByPhone.cshtml.cs (hunk 2 above)
Comment (Top-Cat) [reply src/ui/connect-ui/src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/ContactByPhone.cshtml.cs:25]: I should put in a ticket to just replace a load of these so you don't have to comment this on every PR 🤣

**Review by Zac-Digital — APPROVED** [review]

**Review by StuwiiDev — APPROVED** [review]

**Pull request merged**